### ISSUE TITLE
Template List File: One sample per running ccne-fast 

### ISSUE THREAD

**Issue by ne1al** [biojiang/ccne#1]
Hello,
I created a list file as guided in the template your provided but I added more than one sample in the file. Unfortunately, ccne did not finish with the below error. However, when I added only one sample, it ran well. Any advice?
`ccne-fast --cpus 4 --amr NDM-5 --sp Kpn --in all_samples.list --out all_CN.txt`

Error: `not found!./S19_R1.fq.gz or ./S19_R2.fq.gz`

**Comment by biojiang** [comment]
Did S19_R1.fq.gz or S19_R2.fq.gz in the fold you are running?

**Comment by ne1al** [comment]
Yes, in the same folder. The list works only if it has one sample written in it.

**Comment by biojiang** [comment]
This may be caused by the line break as Windows and Linux use different line break. Try to modify the example sample list as an input.
